src/main/java/org/sagebionetworks/bridge/validators/ValidatorUtils.java

```diff
@@ -72,6 +73,13 @@ static void password(Errors errors, PasswordPolicy passwordPolicy, String passwo
             }
         }
     }
+    
```

Comment:
This is tested through AdminAccountValidatorTest. But it might be worth it to add some tests to ValidatorUtilsTest in case the method gets used somewhere else in the future.

Reply:
Definitely, added tests